### PR TITLE
[CNSMR-1987] Outline for Chat Architecture article

### PR DESCRIPTION
**Pull request by mluisbrown**
This is the outline for the Chat Architecture article. It's a very brief outline of what the article will contain.

~I'm creating it as a Draft PR~ Oops, clicked Create PR and now I can't make it a Draft 🤷‍♂ ...anyway this PR will become the article itself. Creating initial PR to get approval of the outline from the editor: @AnnKatF

### CODE REVIEW

File: Cookbook/Technical-Documents/ChatArchitecture.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+### Chat SDK
+
+- Explanation of how the Chat SDK works and an overview of the Chat V3 API
+- Explanation of the concepts of Incoming and Outgoing messages
+- Explanation of the concetps of chat “content” and chat “input”
+
+
+### Chat View Models
+
+Explanation of how amd why chat is broken into 3 separate view models.
```
Comment (AnnKatF) [inline Cookbook/Technical-Documents/ChatArchitecture.md]:
```suggestion
Explanation of how and why chat is broken into 3 separate view models.
```

File: Cookbook/Technical-Documents/ChatArchitecture.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+- Explain responsibilities of the `ChatViewModel`
```
Comment (AnnKatF) [inline Cookbook/Technical-Documents/ChatArchitecture.md]:
```suggestion
- Explain the responsibilities of the `ChatViewModel` 
```

**Review by AnnKatF — APPROVED** [review]
Looks good 👍 
Just some nits.

**Comment by RuiAAPeres** [comment]
Closing due to inactivity. Please re-open when you have time to work on it. ❤️

**Pull request closed without merging**